### PR TITLE
chore: fix up API prefixes to match other SDKs

### PR DESCRIPTION
**Pull request by andrewjl-mux**
* deprecate and keep existing `MuxImaListener` type, forward all calls to it to new API transparently

### CODE REVIEW

File: Sources/MuxStatsGoogleIMAPlugin/include/MuxImaListener+Private.h [hunk 1]
```diff
@@ -1,13 +1,10 @@
 //
-//  Header.h
-//  
+//  MuxImaListener+Private.h
```
Comment (daytime-em) [inline Sources/MuxStatsGoogleIMAPlugin/include/MuxImaListener+Private.h:2]: Was this file intended to be deleted? Since we have `MUXSDKIMAAdsListener+Private.h`

File: Sources/MuxStatsGoogleIMAPlugin/include/module.modulemap [hunk 2]
```diff
@@ -1,8 +1,10 @@
 
 module MuxStatsGoogleIMAPlugin {
     header "MuxImaListener.h"
-    
+    header "MUXSDKIMAAdsListener.h"
+
     explicit module Private {
         header "MuxImaListener+Private.h"
```
Comment (daytime-em) [inline Sources/MuxStatsGoogleIMAPlugin/include/module.modulemap:7]: If `MuxImaListener+Private.h` was renamed, we should remove the old declaration

**Review by daytime-em — CHANGES_REQUESTED** [review]
Looks like evidence of a gnarly rebase

Other than that it looks much more idiomatic, 🙌

File: Sources/MuxStatsGoogleIMAPlugin/include/MuxImaListener+Private.h (hunk 1 above)
Comment (andrewjl-mux) [reply Sources/MuxStatsGoogleIMAPlugin/include/MuxImaListener+Private.h:2]: `MuxImaListener` is now deprecated as of this release. It's intended to be kept around to help transition to the new API for one or two releases. All the calls pass through to the new type. [See here](https://github.com/muxinc/mux-stats-google-ima/pull/44/files#diff-f34e962d1c12ce1a40baede891e9595ae975a59356c44e7cc29eda19317552b4).

Not strongly opposed to removing it, though.

Comment (daytime-em) [reply Sources/MuxStatsGoogleIMAPlugin/include/MuxImaListener+Private.h:2]: Ohh, I see now. Somehow I missed the annotation on `MuxImaListener.h`

Never mind, I'm good with keeping this

**Review by daytime-em — APPROVED** [review]
LGTM, thanks for the explanation!

**Comment by andrewjl-mux** [comment]
Rebased against `releases/v0.14.0` to fix merge conflict by a recent commit.
